src/components/FeatureFlagRenderer.js

```diff
@@ -55,11 +50,10 @@ export default class FeatureFlagRenderer extends Component {
   }
 
   _checkFeatureFlag () {
-    const { launchDarklyConfig: { clientId, user }, flagKey } = this.props;
-    const ldClient = ldBrowserInit(clientId, user);
+    const { ldClientWrapper, flagKey } = this.props;
```

Comment:
disregard, I should have reread the manual.  We already get that for free from `FeatureFlag`.